introduction.md

```diff
@@ -4,3 +4,12 @@ Hi,
 I'm Passionate developer, always looking forward to learn new things and eager to change people's lives with a piece of software.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?